### PR TITLE
Thoughts for after #1020

### PR DESCRIPTION
**Pull request by trask**
Hi all,

I just wanted to share additional thoughts here for PR(s) after #1020. I'm traveling this week, so won't have much time to address things until Fri, but let me know what you think.

Thanks,
Trask

### CODE REVIEW

**Comment by trask** [comment]
Btw the thought behind the last two (organize imports and formatting) is just to lessen merge conflicts going forward once we start OpenTelemetry work.

**Comment by trask** [comment]
rebased on top of #1020

**Comment by trask** [comment]
Rebased again

File: dd-java-agent/instrumentation/spring-webflux/src/main/java8/datadog/trace/instrumentation/springwebflux/client/DefaultWebClientAdvice.java [hunk 1]
```diff
@@ -10,18 +10,21 @@
...
-      && !clientRequest.headers().keySet().contains("x-datadog-trace-id")) {
+        && mono != null
+        // The response of the
+        // org.springframework.web.reactive.function.client.ExchangeFunction.exchange method is
+        // replaced by a decorator that in turn also calls the
+        // org.springframework.web.reactive.function.client.ExchangeFunction.exchange method. If the
+        // original return value
+        // is already decorated (we detect this if the "x-datadog-trace-id" is added), the result is
+        // not decorated again
+        // to avoid StackOverflowErrors.
```
Comment (tylerbenson) [inline dd-java-agent/instrumentation/spring-webflux/src/main/java8/datadog/trace/instrumentation/springwebflux/client/DefaultWebClientAdvice.java]: Maybe change this to a block comment instead?

**Comment by tylerbenson** [comment]
@trask When you're ready for this to be reviewed, please rebase off of master now that the other PR is merged and mark "ready for review".

**Comment by trask** [comment]
Rebased, let me know if you want me to pull any of this out into a separate PR.

File: dd-java-agent/instrumentation/elasticsearch/rest-5/src/latestDepTest/groovy/Elasticsearch6RestClientTest.groovy [hunk 2]
```diff
@@ -51,11 +51,11 @@ class Elasticsearch6RestClientTest extends AgentTestRunner {
...
-      @Override
-      RequestConfig.Builder customizeRequestConfig(RequestConfig.Builder builder) {
-        return builder.setConnectTimeout(Integer.MAX_VALUE).setSocketTimeout(Integer.MAX_VALUE)
-      }
-    })
+        @Override
+        RequestConfig.Builder customizeRequestConfig(RequestConfig.Builder builder) {
+          return builder.setConnectTimeout(Integer.MAX_VALUE).setSocketTimeout(Integer.MAX_VALUE)
+        }
+      })
```
Comment (tylerbenson) [inline dd-java-agent/instrumentation/elasticsearch/rest-5/src/latestDepTest/groovy/Elasticsearch6RestClientTest.groovy]: Do you know of any changes we can make to the build scripts to automatically enforce this better?  Unfortunately google java format only works for java.

Comment (trask) [reply dd-java-agent/instrumentation/elasticsearch/rest-5/src/latestDepTest/groovy/Elasticsearch6RestClientTest.groovy]: No, not sure how to enforce groovy format.

File: dd-java-agent/instrumentation/ratpack-1.4/src/test/groovy/RatpackOtherTest.groovy [hunk 3]
```diff
@@ -129,14 +132,13 @@ class RatpackOtherTest extends AgentTestRunner {
...
-          .startActive(true)
+      final AgentSpan span =
+        startSpan("ratpack.exec-test")
+          .setTag(DDTags.RESOURCE_NAME, "INSIDE-TEST")
+      final AgentScope scope = activateSpan(span, true)
 
-      ((TraceScope) scope).setAsyncPropagation(true)
-      scope.span().setBaggageItem("test-baggage", "foo")
+      scope.setAsyncPropagation(true)
+      span.setBaggageItem("test-baggage", "foo")
```
[the remark this answers is not in the document]
Comment (trask) [reply dd-java-agent/instrumentation/ratpack-1.4/src/test/groovy/RatpackOtherTest.groovy]: Makes sense, I'll make this change.

**Comment by tylerbenson** [comment]
Oops, sorry for the conflicts.  LMK if you need help resolving.

**Comment by trask** [comment]
Rebased, and addressed issues we discussed

**Pull request merged**